### DISCUSSION TITLE
Changing uri for multiple crazyradio 2.0

### DISCUSSION THREAD

**Discussion by adarsh2798** [bitcraze/discussions#1180] [Q&A]
Hi,

I am new to crazyflie. I want to control about 10 crazyflies and hence i plan to use 2 crazyradios. Both my crazyradios have been setup by installing the firmware into it. I Can connect to crazyflies using both of them individually on cfclient. For both radios when i plug in, on cflcient i see 'radio://0/80/2M/E7E7E7E701' in scan tab. How to change the first number of uri to 1 for other crazyradio?

**Comment by knmcguire** [comment]
this discussion seems to be closed. Does that you were able to figure it out? If so, it is better to update the thread and select that as an answer.
